_projects/02-foomatic.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Foomatic"
+---
+Foomatic is a database-driven system for integrating free software drivers with common spoolers under UNIX. Foomatic's database contains information to create PPD files for a printer. Foomatic is now maintained only for CUPS and driverless printing. 
```

Comment:
Replace this line by
```
Foomatic is a database-driven system for integrating free software printer drivers with the CUPS printing system and also with an easy-to-use command-line interface for direct, spooler-less printing.
```
Note that driverless printing has nothing to do with drivers.

Reply:
Done.